pkg/trace/force_sampler.go

```diff
@@ -38,3 +47,12 @@ func forceTracing(ctx context.Context) context.Context {
 	}
 	return ctx
 }
+
+// sampleAt sets the sample rate for a given trace explicitly. The rate given will be
+// parsed as 1/rate to actually determine the sample rate.
```

Comment:
nit: the wording here (probably copied from my own from elsewhere) can be improved a little.

```suggestion
// sampleAt sets the sample rate for a given trace explicitly.  
// The rate specifies how many samples were looked at for each 
// accepted sample.  That is, the actual sampling  = 1/rate.
```